Gemfile

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+source "https://rubygems.org"
+
+git_source(:github) {|repo_name| "https://github.com/#{repo_name}" }
+gem "rspec"
+# gem "rails"
```

Comment:
Config your editor to avoid not adding a new line at the end of file. There's a good explanation for adding a new line at the end of the file [here](https://unix.stackexchange.com/a/18789)